azurelinuxagent/common/protocol/restapi.py

```diff
@@ -65,8 +65,9 @@ def __init__(self):
 
 
 class VMAgentManifest(object):
-    def __init__(self, family):
+    def __init__(self, family, version=None):
         self.family = family
+        self.version = version
```

Comment:
is this the requested version? can we document it?

Reply:
Yes, let me add a comment here

Reply:
Done

Reply:
Thanks for adding that... I feel None may not be the best choice for this. Callers need to be aware that this property can be None and protect themselves against (e.g. if at some point somebody tries to create a Flexible Version out of this property). Maybe a special value (say 0.0.0.0) would be more appropriate?

Reply:
Done